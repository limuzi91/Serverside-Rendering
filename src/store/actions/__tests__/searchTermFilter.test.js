import { setSearchTermFilter, restFilters } from "../index";

it("test setSearchTermFilter through the Snapshot approach", () => {
  expect(setSearchTermFilter("New York")).toMatchSnapshot();
});

it("test restFilters through the Snapshot approach", () => {
  expect(restFilters()).toMatchSnapshot();
});
