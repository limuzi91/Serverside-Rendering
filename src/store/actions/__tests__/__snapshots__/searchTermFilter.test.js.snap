// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test restFilters through the Snapshot approach 1`] = `
Object {
  "type": "RESET_FILTERS",
}
`;

exports[`test setSearchTermFilter through the Snapshot approach 1`] = `
Object {
  "payload": "New York",
  "type": "SET_TEXT_FILTER",
}
`;
